sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -1580,6 +1584,38 @@ def test_split_half(self):
     raise unittest.SkipTest("This test is for a single worker only.")
 
 
+class FnApiBasedLullLoggingTest(unittest.TestCase):
+  def create_pipeline(self):
+    return beam.Pipeline(
+        runner=fn_api_runner.FnApiRunner(
+            default_environment=beam_runner_api_pb2.Environment(
+                urn=python_urns.EMBEDDED_PYTHON_GRPC),
+            progress_request_frequency=0.5))
+
+  def test_lull_logging(self):
+
+    if sys.version_info < (3, 4):
```

Comment:
SDK is not installable with < Python 3.4 anyway. You can probably drop this (https://github.com/apache/beam/blob/master/sdks/python/setup.py#L182)

Or is this for protecting against py 2 users?

Reply:
This is also for Python 2, that's right.

Reply:
Ack. You could add a BEAM-1251 todo here so that we can remove it after python 2 support is removed.